lib/graphql/schema/subscription.rb

```diff
@@ -92,6 +92,25 @@ def load_application_object_failed(err)
       def unsubscribe
         raise UnsubscribedError
       end
+
+      # Call this method to provide a new subscription_scope; OR
+      # call it without an argument to get the subscription_scope
+      # @param new_scope [Symbol]
+      # @return [Symbol]
+      def self.subscription_scope(new_scope = nil)
+        if new_scope
+          @subscription_scope = new_scope
+        else
+          @subscription_scope || find_inherited_method(:subscription_scope, nil)
+        end
+      end
```

Comment:
This won't support _overriding_ an inherited scope with `nil`. Do you want to? Here's an implementation that would support overriding with `nil`: 

```ruby 
READING_SCOPE = Object.new 
def self.subscription_scope(new_scope = READING_SCOPE) 
  if new_scope != READING_SCOPE 
    @subscription_scope = new_scope 
  elsif defined?(@subscription_scope) 
    @subscription_scope # This might return `nil` if it was explicitly overridden 
  else 
    find_inherited_method(:subscription_scope, nil) 
  end 
end 
```

Reply:
I didn't consider that, makes a lot of sense. Based it on the implementation of similar fields in the `Resolver class` but I like this implementation better. Will add this in along with some more tests 👌